go-controller/pkg/metrics/node.go

```diff
@@ -15,7 +15,7 @@ var MetricCNIRequestDuration = prometheus.NewHistogramVec(prometheus.HistogramOp
 	Subsystem: MetricOvnkubeSubsystemNode,
 	Name:      "cni_request_duration_seconds",
 	Help:      "The duration of CNI server requests.",
-	Buckets:   prometheus.ExponentialBuckets(.1, 2, 15)},
+	Buckets:   prometheus.LinearBuckets(1, 1, 15)},
```

Comment:
Is your feedback about typical CNI duration times tested only on centralized OVN mode? I'm worried we might be losing some resolution stepping by a second. Is there a performance cost to adding more buckets? Can we make it like .25 step or .5 step?

Reply:
Yeah

Reply:
@trozet got back this answer

For every bucket prometheus keeps time-series of following two values:
1. count of number of entities
2. sum of number of entities

With 0.5 sec step from 1 to 15.. we have about 30 entities.. so, we will need around 60 time-series data. Since we are going to have this metric on only ovnkube-master, it shouldn't be a big issue.

So, I am going to change the step to 0.5 secs.